plugins/Rockets/SnapshotTask.h

```diff
@@ -88,18 +105,17 @@ class SnapshotFunctor : public TaskFunctor
         else
             *_camera = engine.getCamera();
 
-        *_scene = engine.getScene();
+        _scene->cloneFrom(engine.getScene());
     }
 
     ImageGenerator::ImageBase64 operator()()
     {
+        _scene->commit();
```

Comment:
Any reason for this reordering?

Reply:
To comply what the 'normal' brayns rendering loop does, otherwise no real reason that would fix anything.